cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -87,9 +87,9 @@ class GetAllWork : public PooledWork<GfErrType>,
         m_keys(keys),
         m_region(region),
         m_aCallbackArgument(aCallbackArgument) {
-    m_request = new TcrMessageGetAll(region->getCache().createDataOutput(),
-                                     region.get(), m_keys.get(), m_poolDM,
-                                     m_aCallbackArgument);
+    m_request = new TcrMessageGetAll(
+        new DataOutput(region->getCache().createDataOutput()), region.get(),
```

Comment:
Don't get the DataOutput from Cache, use:
```
m_connManager.getCacheImpl()->createDataOutput()
```

Reply:
This is a method on the GetAllWork class, not ThinClientPoolDM, so it doesn't have an m_connManager.

Reply:
👍